js_modules/dagit/packages/core/src/instance/InstanceTabs.tsx

```diff
@@ -31,7 +31,7 @@ export const InstanceTabs = <TData extends Record<string, any>>(props: Props<TDa
           to="/code-locations"
           icon={<WorkspaceStatus placeholder={false} />}
         />
-        <TabLink id="workspace" title="Workspace" to="/workspace" />
+        <TabLink id="definitions" title="Definitions" to="/workspace" />
```

Comment:
Instead of "Workspace", use "Definitions". This tab will be removed altogether soon, in favor of Josh's new design for the Code Locations tab.